.cirrus.yml

```diff
@@ -104,7 +104,6 @@ gating_task:
         # N/B: entrypoint.sh resets $GOSRC (same as make clean)
         - '/usr/local/bin/entrypoint.sh install.tools |& ${TIMESTAMP}'
         - '/usr/local/bin/entrypoint.sh validate |& ${TIMESTAMP}'
-        - '/usr/local/bin/entrypoint.sh golangci-lint |& ${TIMESTAMP}'
```

Comment:
Would it make sense then to also remove `hack/man-page-checker` from line 113 (below, under `build_script`) ?

Reply:
sure!